contracts/token/ERC6909/extensions/draft-ERC6909ContentURI.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ERC6909} from "../draft-ERC6909.sol";
+import {IERC6909ContentURI} from "../../../interfaces/draft-IERC6909.sol";
+
+/**
+ * @dev Implementation of the Content URI extension defined in ERC6909.
+ */
+contract ERC6909ContentURI is ERC6909, IERC6909ContentURI {
+    string private _contractURI;
+    mapping(uint256 id => string) private _tokenURIs;
+
+    /// @inheritdoc IERC6909ContentURI
+    function contractURI() external view virtual override returns (string memory) {
+        return _contractURI;
+    }
```

Comment:
Should we emit the `ContractURIUpdated()` event? 

https://eips.ethereum.org/EIPS/eip-7572